chapter01/sxg/HelloWorld/pom.xml

```diff
@@ -0,0 +1,45 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.github.sebin</groupId>
+  <artifactId>HelloWorld</artifactId>
+  <packaging>war</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>HelloWorld Maven Webapp</name>
+  <url>http://maven.apache.org</url>
+ <properties>
+		<spring.version>4.0.1.RELEASE</spring.version>
```

Comment:
...but it looks so much nicer in a property like this, and for that you will be forgiven :smile: